src/main/java/com/conveyal/datatools/manager/models/FeedSourceSummary.java

```diff
@@ -0,0 +1,548 @@
+        );
+        return extractFeedSourceSummaries(projectId, stages);
+    }
+
+    /**
+     * Get the latest feed version from all feed sources for this project.
+     */
+    public static Map<String, FeedVersionSummary> getLatestFeedVersionForFeedSources(String projectId) {
+        /*
+            Note: To test this script:
+                1) Comment out the call to tearDownDeployedFeedVersion() in FeedSourceControllerTest -> tearDown().
+                2) Run FeedSourceControllerTest to created required objects referenced here.
+                3) Once complete, delete documents via MongoDB.
+                4) Uncomment the call to tearDownDeployedFeedVersion() in FeedSourceControllerTest -> tearDown().
+                5) Re-run FeedSourceControllerTest to confirm deletion of objects.
+
+            db.getCollection('FeedSource').aggregate([
+                {
+                    // Match provided project id.
+                    $match: {
+                        projectId: "project-with-latest-deployment"
+                    }
+                },
+                {
+                    $lookup: {
+                        from: "FeedVersion",
+                        localField: "_id",
+                        foreignField: "feedSourceId",
+                        as: "feedVersions"
+                    }
+                },
+                {
+                    $unwind: "$feedVersions"
+                },
+                {
+                    $group: {
+                        _id: "$_id",
+                        doc: {
+                            $max: {
+                                version: "$feedVersions.version",
+                                feedVersionId: "$feedVersions._id",
+                                firstCalendarDate: "$feedVersions.validationResult.firstCalendarDate",
+                                lastCalendarDate: "$feedVersions.validationResult.lastCalendarDate",
+                                issues: "$feedVersions.validationResult.errorCount"
+                            }
+                        }
+                    }
+                }
+            ])
+        */
+        List<Bson> stages = Lists.newArrayList(
+            match(
+                in("projectId", projectId)
+            ),
+            lookup("FeedVersion", "_id", "feedSourceId", "feedVersions"),
+            unwind("$feedVersions"),
+            group(
+                "$_id",
+                Accumulators.first("feedVersionId", "$feedVersions._id"),
+                Accumulators.first("firstCalendarDate", "$feedVersions.validationResult.firstCalendarDate"),
+                Accumulators.first("lastCalendarDate", "$feedVersions.validationResult.lastCalendarDate"),
+                Accumulators.first("errorCount", "$feedVersions.validationResult.errorCount")
+            )
+        );
+        return extractFeedVersionSummaries(
+            "FeedSource",
+            "feedVersionId",
+            "_id",
```

Comment:
Yep! The values to be extracted are different here because it's working with feed source documents. The feed source key in this case is the document id (_id) and not a document field (in the case of the other calls).